seshdash/tasks.py

```diff
@@ -109,6 +110,13 @@ def get_BOM_data():
             if v_client.IS_INITIALIZED:
                         bat_data = v_client.get_battery_stats(int(site.vrm_site_id))
                         sys_data = v_client.get_system_stats(int(site.vrm_site_id))
+
```

Comment:
this code here was removed in a recent commit so it shouldn't be here. Please pull from master first.